spec/features/record_phone_call_activity_spec.rb

```diff
@@ -31,53 +31,33 @@
     expect_to_be_on_record_phone_call_page
 
     # Test required fields
-    click_button "Continue"
+    click_on "Add phone call"
```

Comment:
I think it's helpful to assert that it's a button rather than a link?

```suggestion
    click_button "Add phone call"
```